features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Documents/Listing.jsx

```diff
@@ -121,7 +121,7 @@ class Listing extends React.Component {
                     'Error[' + errorData.code + ']: ' + errorData.description + ' | ' + errorData.message + '.';
                 console.error(messageTxt);
                 Alert.error(intl.formatMessage({
-                    id: 'Apis.Details.Documents.Listing.documents.listing.fetching.error.message',
+                    id: 'Apis.product.Details.Documents.Listing.documents.listing.fetching.error.message',
```

Comment:
not into the localization Id format we agreed.

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/commit/7e7dde3ab4c9d9678e8f63d948920b9c7ab6d0bc#diff-aece198b4ce15ccec4d03c8282220531R124